test/specs/spellchecker_test.js

```diff
@@ -37,6 +37,9 @@ describe('main/Spellchecker.js', function() {
       SpellChecker.getSpellCheckerLocale('ru').should.equal('ru-RU');
       SpellChecker.getSpellCheckerLocale('ru-RU').should.equal('ru-RU');
 
+      SpellChecker.getSpellCheckerLocale('sv').should.equal('sv-SE');
+      SpellChecker.getSpellCheckerLocale('sv-SE').should.equal('sv-SE');
+
```

Comment:
can you also add a sample of spellchecking corrections that should work with this language? see line 45 and below for examples of it.

Reply:
Thanks for the info about the change in spellchecker and I hope that Swedish spellchecking also makes into your new “big change”. We are using Mattermost at work and we got annoyed using the desktop version with Swedish text (works fine in browser).  Saw that 5.32 server got Swedish as supported language great news! I have added test for sv-SE but circleci test seems to been stopped for some other reason than my changes.

Reply:
yep, sometimes a couple of test might fail. I'm rerunnintg them now to see if that passes.